user/tests/wiring/no_fixture/cellular.cpp

```diff
@@ -190,16 +205,17 @@ test(MDM_01_socket_writes_with_length_more_than_1023_work_correctly) {
     int res = c.connect("httpbin.org", 80);
     (void)res;
 
-    int sz = c.write((const uint8_t*)request, requestSize);
+    int sz = c.write((const uint8_t*)request.get(), requestSize);
     assertEqual(sz, requestSize);
 
-    char* responseBuf = new char[2048];
-    memset(responseBuf, 0, 2048);
+    auto responseBuf = std::make_unique<char[]>(bufferSize);
+    assertTrue((bool)responseBuf);
+    memset(responseBuf.get(), 0, bufferSize);
     int responseSize = 0;
     uint32_t mil = millis();
     while(1) {
         while (c.available()) {
```

Comment:
What to do if responseSize >= bufferSize?

Reply:
Fixed.